src/bindgen/ir/function.rs

```diff
@@ -166,6 +168,15 @@ impl Source for Function {
                     write!(out, "{}", postfix);
                 }
             }
+
+            let attributes = func
+                .attributes
+                .iter()
+                .map(|s| format!(" __attribute__(({}))", s))
```

Comment:
We've generally avoided hardcoding attribute names and `__attribute__` declarations here. I think a slightly more consistent approach would be something like an `[fn] swift_name_macro = "FOO"` config. This would get the raw content of the string just like `CF_SWIFT_NAME` does. So that to use this feature the only thing you need to do would be `[fn] swift_name_macro = "CF_SWIFT_NAME"`, for example.

Reply:
Yes, that's a much better solution. It doesn't work quite so well for the `__attribute__` version of this behaviour because of the double-parentheses needed, but most users will probably prefer `CF_SWIFT_NAME` or similar instead (and a macro can be defined and included separately if necessary).